libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/helpers/JsExport.kt

```diff
@@ -0,0 +1,42 @@
+package com.apollographql.apollo3.compiler.codegen.kotlin.helpers
+
+import com.apollographql.apollo3.compiler.applyIf
+import com.apollographql.apollo3.compiler.codegen.kotlin.KotlinContext
+import com.apollographql.apollo3.compiler.codegen.kotlin.KotlinSymbols
+import com.apollographql.apollo3.compiler.codegen.maybeFlatten
+import com.apollographql.apollo3.compiler.ir.IrModel
+import com.apollographql.apollo3.compiler.ir.IrModelGroup
+import com.squareup.kotlinpoet.AnnotationSpec
+import com.squareup.kotlinpoet.TypeSpec
+
+internal val suppressNonExportableType = AnnotationSpec.builder(KotlinSymbols.Suppress)
+  .addMember("%S", "NON_EXPORTABLE_TYPE")
```

Comment:
Can you add a comment there? My understanding is that we're making the top level query exportable but what we actually want is the nested models to be exportable. The query being exported is just a side effect of Kotlin not supporting the annotation on nested classes?